frontend/src/actions/github/fetch_repos.js

```diff
@@ -0,0 +1,40 @@
+import { getThirdParty } from '../../lib/api';
+import * as constants from '../../constants/github';
+
+
+/**
+ * Fetch a user's repo
+ */
+
+export default (username) => {
```

Comment:
These action files are very repetitive/copy-pasted, have a generic structure of request/success/failure. Might benefit from e.g a prototype or template.